frontend/tampere/enCustomizations.tsx

```diff
@@ -267,10 +267,10 @@ const en: DeepPartial<Translations> = {
         },
         assistanceNeedInstructions: {
           DAYCARE:
-            'Intensified or special care is given to a child as soon as the need arises. If a child has received an expert opinion backing the need for support, this must be notified in the early childhood education application. The support measures are carried out in the child’s daily life as part of the early childhood educational activities. Tampere’s early childhood education will separately be in contact after the application has been submitted, if the child has a need for support.',
+            'Intensified or special care is given to a child as soon as the need arises. If a child has received an expert opinion backing the need for support, this must be notified in the early childhood education application. The support measures are carried out in the child’s daily life as part of the early childhood educational activities. Tampere’s early childhood education will separately be in contact after the application has been submitted, if the child has a need for support. A place arrangement requires that additional information is shown for early childhood education service guides and a special early childhood education teacher.',
```

Comment:
The English translation could use some work. I'm not sure if we've established a general English term for "palveluohjaus" but the one used here seems off. I think the users are called "service workers" in the application parlance, but that sounds weird as well. 

The special education teacher should not be referred to with an indefinite article. If it's important to communicate that the information will be seen by only one teacher then maybe "your child's special early childhood education teacher". If the information is visible to some unknown teachers then use plural.

If the "additional information" in the info text refers to something the user has elected to enter on the same form, it might be a good idea to be more clear about that e.g. "the entered additional information", "this additional information" or maybe just "the additional information".